src/WWT.Azure/WWT.Azure.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
```

Comment:
Only need netstandard here. netstandard2.0 is the lowest common denominator for .NET 5 and .NET 4.8, so we can just target that. Later on, if it makes sense, it can be switched to net5.0 to enable some new features (can't think of too many right now).